src/Components/IconNotification.vue

```diff
@@ -0,0 +1,91 @@
+<!--
+  - SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<template>
+	<!-- Modified IconBell from material design icons -->
+	<span v-if="showDot || showWarning" class="notifications-button__icon">
```

Comment:
Could you move the comment inside the element? Otherwise this component has kind of 2 root nodesa: a comment node + span/svg node.

It works fine in most of the cases, because the comment is removed, but not always. For example, in Vue 3 and testing.